### PR TITLE
Manage donations

### PR DESCRIPTION
**Pull request by Tony07200**
### Valide adminGiftCrud avant celle ci

Donation manager page

After we saw than we can call the payment method of each gift, we try to put it into the table of gift.
So we fixed some trouble with migration / database.
With the morphism, when we put something with it, it take the real name of Class clased (ex: App\Example) instead of what we thought : 'gift'...

Make : php artisan migrate:fresh --seed (for changes Bdd)

### CODE REVIEW

File: app/Http/Controllers/Admin/GiftController.php [hunk 1]
```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Gift;
+use App\Http\Controllers\Controller;
+use App\User;
+use Auth;
+use Illuminate\Http\Request;
+use phpDocumentor\Reflection\DocBlock\Tags\Author;
```
Comment (RMaxence) [inline app/Http/Controllers/Admin/GiftController.php]: "Import 'phpDocumentor\Reflection\DocBlock\Tags\Author' is never used"

Comment (RMaxence) [reply app/Http/Controllers/Admin/GiftController.php]: (in Admin/GiftController and User/GiftController)

**Comment by RMaxence** [comment]
I can make a negativ donation, I CAN TAKE YOUR MONEY ! :grin: 

Carlee Cole | -2000.00 € | 09/08/2018

**Comment by MichaelJouve** [comment]
for me its good after this last change (negative value)

**Comment by Noksit** [comment]
@RMaxence negativ amount corrected
@anzorissaev maximum 999999€ set. space and word not available

**Pull request merged**